src/metabase/driver/common.clj

```diff
@@ -409,13 +409,18 @@
 (def ^:private ^clojure.lang.PersistentVector days-of-week
   [:monday :tuesday :wednesday :thursday :friday :saturday :sunday])
 
+(def ^:dynamic *start-of-week*
+  "Used to override start-of-week settings."
```

Comment:
This docstring should be a bit better about the circumstances it is intended to be used.